src/modules/course/dto/create-course.dto.ts

```diff
@@ -40,4 +41,21 @@ export class CreateCourseDto {
   @IsNotEmpty()
   @IsArray()
   mentor: [];
+
+  @IsNotEmpty()
+  tags: TagType;
```

Comment:
Tags should be an array of tag. and add enum prop

Reply:
will enum prop work on array ?